linode/manager
# refactor: [M3-6274] -  MUI v5 Migration - `SRC > Features > Longview` Pt1: Longview Detail

## Description 📝
- migrating styling for src > features >longview > longview detail. Splitting this ticket up into two PRs (this one is still 50 changed files though yikes 😅 ) -- this one, which covers LongviewDetail, and a second one, which will cover LongviewLanding -- so that it's easier to test / catch regressions

## Major Changes 🔄
Inside LongivewDetail package, 
- remove React.FC
- change to named exports (unless components were being lazily exported, then kept as is)
- remove makeStyles
- some files had a lot of common styling, so moved all this to CommonStyles.styles.ts (similar to [this ticket](https://github.com/linode/manager/pull/9445) from way back)

## How to test 🧪
- navigate to Longview Details and ensure that everything looks as expected, and that all functionality is still the same
